api/envoy/api/v2/route/route.proto

```diff
@@ -104,6 +104,28 @@ message VirtualHost {
   // specific; see the :ref:`HTTP filter documentation <config_http_filters>`
   // for if and how it is utilized.
   map<string, google.protobuf.Struct> per_filter_config = 12;
+
+  message RetryPriority {
+    string name = 1;
```

Comment:
Add validation that this name is not empty (`[(validate.rules).string.min_bytes = 1]`)